src/common/components/ui/formik/validationSchemas.ts

```diff
@@ -9,7 +9,7 @@ import { ProxyFieldsType } from '../../../types';
 import { trimCommaSeparatedList, trimSshPublicKey } from './utils';
 
 const ALPHANUMBERIC_REGEX = /^[a-zA-Z0-9]+$/;
-const CLUSTER_NAME_REGEX = /^[a-z]([-a-z0-9]*[a-z0-9])?$/;
+const CLUSTER_NAME_REGEX = /^[a-z](.*[a-z0-9])?$/;
```

Comment:
`.*` means that the name can contain special characters in the middle (e.g. $%^&*#...)
Why is this change necessary?

Reply:
I wanted the conditions in the list to be simple, straight forward and always deal with one thing at a time.
This regex was good but it tested two things at a time and that didn't allow for separation of the conditions ("you can use these characters" and "name must start and end with ..."). I think that for the users a slightly longer list with shorter instructions is easier to comprehend than a shorter list with long and complicated instructions.